support-lambdas/acquisitions-firehose-transformer/src/main/scala/com/gu/acquisitionFirehoseTransformer/AcquisitionToJson.scala

```diff
@@ -18,6 +18,7 @@ object AcquisitionToJson {
     countryCode: String,
     amount: BigDecimal,
     annualisedValue: Double,
+    annualisedValueGBP: Double,
```

Comment:
the super mode calculator can be updated to query the new field later

Reply:
just to check - do we think it's best to add this new field, instead of updating the old one?

Reply:
I think so, the name should really have included `GBP` anyway!